templates/http/docs/example.md.twig

```diff
@@ -1,4 +1,4 @@
-{% set boundary = random() %}
+{% set boundary = 'cec8e8123c05ba25' %}
```

Comment:
If the class name is REST, why do we have the templates in HTML directory?

Reply:
Have moved to a rest directory 👌